event-checklist.md

```diff
@@ -0,0 +1,104 @@
+#Event committee checklist
+##Introduction
+Teams often run events under the auspices of [Linux Australia](https://www.linux.org.au). This document articulates the milestones you need to check off to ensure compliance with Linux Australia policies, good operations management - and to deliver a great event.
+
+##Overview
+The steps can be summarised as;
+1. Form an event subcommittee under the [event subcommittee policy](subcommittee_policy_v2.md).  
+2. Understand financial arrangements and set up banking access
+3. Put in place event infrastructure
+4. Put in place event Code of Conduct
+5. Ensure insurance coverage for event
+6. Ensure recognition of Linux Australia in event's digital presence
+7. Ensure Sponsorship Agreement is in place
+8. Ensure Post Event Report is written and distributed
+
+##Checklist for Event Teams
+* [ ] Get your event team together and ensure you have a balanced mix of skills
+* [ ] Read through the [event subcommittee policy](subcommittee_policy_v2.md)
+* [ ] Ensure all members of your team are [members of Linux Australia](https://www.linux.org.au/membership)
+* [ ] Raise a motion to form a new subcommittee by [emailing the Council](mailto:council@linux.org.au). A draft motion is provided below.
+* [ ] Understand expectations from Linux Australia regarding the percentage of event profits that are expected to be returned to Linux Australia

```

Comment:
err all of it.. I'd drop the percentage part to make it clear that LA retains all surplus. This item should be more about having an understanding of how LA uses surplus from events and that we're all about supporting the wider open source community etc .